<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-R9JNNG2" failures="2" tests="2" name="AddCustomer" time="19.824" errors="0" timestamp="2023-02-05T21:43:47 IST">
  <testcase classname="testcases.AddCustomerTest" name="addCustomer" time="1.604">
    <failure type="java.lang.AssertionError" message="failed to add customer">
      <![CDATA[java.lang.AssertionError: failed to add customer
at testcases.AddCustomerTest.addCustomer(AddCustomerTest.java:85)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- addCustomer -->
  <testcase classname="testcases.AddCustomerTest" name="addCustomer" time="10.729">
    <failure type="java.lang.AssertionError" message="failed to add customer">
      <![CDATA[java.lang.AssertionError: failed to add customer
at testcases.AddCustomerTest.addCustomer(AddCustomerTest.java:85)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- addCustomer -->
</testsuite> <!-- AddCustomer -->
